compiler/rustc_infer/src/infer/error_reporting/mod.rs

```diff
@@ -1782,7 +1839,8 @@ impl<'a, 'tcx> InferCtxt<'a, 'tcx> {
         self.tcx.note_and_explain_type_err(diag, terr, cause, span, body_owner_def_id.to_def_id());
 
         if let Some(ValuePairs::PolyTraitRefs(exp_found)) = values
-            && let ty::Closure(def_id, _) = exp_found.expected.skip_binder().self_ty().kind()
+            &&
+        let ty::Closure(def_id, _) = exp_found.expected.skip_binder().self_ty().kind()
```

Comment:
there are a lot of seemingly unrelated formatting changes here. is there a reason for them?

Reply:
Should be better now.